sympy/solvers/ode/tests/test_single.py

```diff
@@ -163,6 +163,13 @@ def _test_all_hints(runxfail=False):
         _test_all_examples_for_one_hint(our_hint, all_examples, runxfail)
 
 
+def _test_dummy_sol(expected_sol,dsolve_sol):
+    if type(dsolve_sol)==list:
+        return any(expected_sol.dummy_eq(sub_dsol) for sub_dsol in dsolve_sol)
```

Comment:
We need to confirm that all solutions from expected_sol match something from dsolve_sol and vice versa.

Reply:
yes it is tested.